src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

```diff
@@ -0,0 +1,68 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<lifecycleMappingMetadata>
+    <pluginExecutions>
+        <pluginExecution>
+            <pluginExecutionFilter>
+                <goals>
+                    <goal>copy</goal>
+                    <goal>copy-dependencies</goal>
+                    <goal>properties</goal>
+                    <goal>unpack</goal>
+                    <goal>unpack-dependencies</goal>
+                </goals>
+            </pluginExecutionFilter>
+            <action>
+                <execute>
+                    <runOnIncremental>true</runOnIncremental>
+                    <runOnConfiguration>false</runOnConfiguration>
+                </execute>
+            </action>
+        </pluginExecution>
+        <pluginExecution>
+            <pluginExecutionFilter>
+                <goals>
+                    <goal>analyze</goal>
+                    <goal>analyze-dep-mgmt</goal>
+                    <goal>analyze-only</goal>
+                    <goal>analyze-report</goal>
+                    <goal>analyze-duplicate</goal>
+                    <goal>build-classpath</goal>
+                    <goal>display-ancestors</goal>
+                    <goal>get</goal>
+                    <goal>go-offline</goal>
+                    <goal>list</goal>
+                    <goal>list-classes</goal>
+                    <goal>list-repositories</goal>
+                    <goal>purge-local-repository</goal>
+                    <goal>resolve</goal>
+                    <goal>resolve-plugins</goal>
+                    <goal>sources</goal>
+                    <goal>tree</goal>
+                </goals>
+            </pluginExecutionFilter>
+            <action>
+                <execute>
+                    <runOnIncremental>false</runOnIncremental>
+                    <runOnConfiguration>false</runOnConfiguration>
+                </execute>
```

Comment:
```suggestion
                <ignore/>
```
Do you want to ignore the execution? Or just want to run it on full or clean builds? Because the latter is what the current configuration is doing.

Reply:
No longer relevant as m2e metadata has been removed from this PR, but for the record, I still want it to be run with full/clean builds.